packages/builder/src/components/common/LinkedRecordSelector.svelte

```diff
@@ -0,0 +1,74 @@
+<script>
+  import { onMount } from "svelte"
+  import { backendUiStore } from "builderStore"
+  import api from "builderStore/api"
+
+  export let modelId
+  export let linkName
+  export let linked = []
+
+  let records = []
+
+  async function fetchRecords() {
+    const FETCH_RECORDS_URL = `/api/${$backendUiStore.selectedDatabase._id}/${modelId}/records`
```

Comment:
url has changed in master

Reply:
also... we will probably want pagination, tho maybe this is not required for now.

Reply:
Yep I think we will want server side pagination. Right now it's all just done client side